examples/webgl_modifier_subdivision.html

```diff
@@ -173,22 +173,28 @@
 				 	'<br><br>Subdivisions: '  + subdivisions +
 					' <a href="#" onclick="nextSubdivision(1); return false;">more</a>/<a href="#" onclick="nextSubdivision(-1); return false;">less</a>' +
 					'<br>Geometry: ' + dropdown + ' <a href="#" onclick="nextGeometry();return false;">next</a>' +
-					'<br><br>Vertices count: before ' + geometry.vertices.length + ' after ' + smooth.vertices.length +
-					'<br>Face count: before ' + geometry.faces.length + ' after ' + smooth.faces.length
+					'<br><br>Vertices count: before ' + geometry.vertices.length + ' after ' + (smooth.attributes.position.array.length/3) +
+					'<br>Face count: before ' + geometry.faces.length + ' after ' + (smooth.attributes.position.array.length / 3)
+                /* since the result is unindex'd the face count will always be directly related to the vertice count */
 				; //+ params.args;
 			}
 
 			function addStuff() {
 
-				if ( cube ) {
-
+				if ( cube != false ) {
+				    
 					scene.remove( group );
-					scene.remove( cube );
+					scene.remove(cube);
+					scene.remove(mesh);
```

Comment:
I think so, it does have a group.add(mesh) in there. was mostly poking in the dark trying to find the thing that was leaking, it was when I did geometry.dispose() that things stopped leaking, so I think that's all that was really required. I don't think the other stuff hurts though, except for making it uglier and longer than necessary.